Packages/com.unity.inputsystem/Documentation~/Processors.md

```diff
@@ -100,18 +100,9 @@ If you [create a layout from JSON](Layouts.md#layout-from-json), you can specify
 
 The Input System package comes with a set of useful Processors you can use.
 
-### Axis Deadzone
-
-|__Name__|`AxisDeadzone`|
-|---|---|
-|__Operand Type__|`float`|
-|__Parameters__|`float min`<br>`float max`|
-
-An axis deadzone Processor scales the values of a Control so that any value with an absolute value smaller than `min` is 0, and any value with an absolute value larger than `max` is 1 or -1. Many Controls don't have a precise resting point (that is, they don't always report exactly 0 when the Control is in the center). Using the `min` value on a deadzone Processor avoids unintentional input from such Controls. Also, some Controls don't consistently report their maximum values when moving the axis all the way. Using the `max` value on a deadzone Processor ensures that you always get the maximum value in such cases.
-
 ### Clamp
 
-|__Name__|`Clamp`|
+|__Name__|[Clamp](../api/UnityEngine.InputSystem.Processors.ClampProcessor.html)|
```

Comment:
You removed the quotes `` here but added them in the later tables. Should this one also have quotes or should the others also have them removed? As its a url I think they probably need removing.

Reply:
Think having the `` makes sense. Fixed.